src/coreclr/debug/di/divalue.cpp

```diff
@@ -4420,14 +4420,7 @@ HRESULT CordbHandleValue::Dispose()
                           m_appdomain->GetADToken());
 
     event.DisposeHandle.vmObjectHandle = vmObjHandle;
-    if (m_handleType == HANDLE_STRONG)
-    {
-        event.DisposeHandle.fStrong = TRUE;
-    }
-    else
-    {
-        event.DisposeHandle.fStrong = FALSE;
-    }
+    event.DisposeHandle.handleType = m_handleType;
```

Comment:
@hoyosjs FYI, I should have had more curiosity when you asked about deleting pinned handles.  It turns out the implementation layers needed fixes (even though ICorDebug didn't).